src/mbgl/style/style_parser.cpp

```diff
@@ -242,16 +241,21 @@ void StyleParser::parseSources(const JSValue& value) {
     }
 }
 
-std::unique_ptr<mapbox::geojsonvt::GeoJSONVT> StyleParser::parseGeoJSON(const JSValue& value) {
+std::unique_ptr<mapbox::geojsonvt::GeoJSONVT> StyleParser::parseGeoJSON(const JSValue& value, std::unique_ptr<SourceInfo>& info) {
```

Comment:
`std::unique_ptr<SourceInfo>&` ⇢ `SourceInfo&`. There's no ownership transfer happening here, so `std::unique_ptr` should not be used.